CHANGELOG.md

```diff
@@ -43,6 +42,7 @@ x.y.z Release notes (yyyy-MM-dd)
   assertion failure rather than a more appropriate exception.
   ([Core #4114](https://github.com/realm/realm-core/issues/4114), since v5.0.0).
 * Fix an issue where calling a MongoDB Realm Function would never be performed as the reference to the weak `User` was lost.
+* Fix error code propagation to object store.
```

Comment:
This doesn't mean anything to the person reading it. The changelog is for developers using the SDK and needs to explain what the change is to them.

Also it's in the wrong section.